x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/search_session_tour.tsx

```diff
@@ -36,13 +40,16 @@ export function useSearchSessionTour(
     if (state === SearchSessionState.Loading) {
       if (!safeHas(storage, TOUR_TAKING_TOO_LONG_STEP_KEY)) {
         timeoutHandle = window.setTimeout(() => {
+          usageCollector?.trackSessionIndicatorTourLoading();
           searchSessionIndicatorRef.openPopover();
         }, TOUR_TAKING_TOO_LONG_TIMEOUT);
       }
     }
 
     if (state === SearchSessionState.Restored) {
+      usageCollector?.trackSessionIsRestored();
```

Comment:
This is probably not the best place to put it because it is part of the tour hook.
It would make more sense to have in inside the  `x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/connected_search_session_indicator.tsx` or `session_service` itself (when `restore`) is called

Reply:
Fixed in: https://github.com/lukasolson/kibana/pull/18